app/controllers/AjaxController.php

```diff
@@ -19,10 +19,20 @@ public function get_data() {
 				$this->email = $_POST['email'];
 				$this->password = $_POST['password'];
 				$this->date = $_POST['date'];
+				$this->mobile = $_POST['mobile'];
 				$this->gender = $_POST['gender'];
 				$this->address = $_POST['address'];
 
-				print_r($this);
+				//print_r($this);
+			}
+
+			public function register() {
+				$user = new User();
+				if ( true === $user->uniquenessChecking($_POST['email']) ) {
+					$user->set_user_data( $_POST );
+					$user->save();
+				}
+				else echo "The email has already been taken.";
```

Comment:
There should be an `exit()` function after echo to terminate the process after ajax response.